tuxemon/states/start/__init__.py

```diff
@@ -121,4 +122,4 @@ def exit_game() -> None:
             label = T.translate(key).upper()
             image = self.shadow_text(label)
             item = MenuItem(image, label, None, callback)
-            self.add(item)
+            self.add(item)
```

Comment:
please avoid changing formatting on source files

Reply:
some editors seem to hate trailing newlines.  its not a big deal, but since this it your first contribution, i just wanted you to know